<ui version="4.0" >
 <class>starterDialog</class>
 <widget class="QDialog" name="starterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Крутильнай маятник</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QPushButton" name="pbStart" >
     <property name="text" >
      <string>Начать эксперимент</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pbSettings" >
     <property name="text" >
      <string>Настройка</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbCalibrateFurnace" >
     <property name="text" >
      <string>Определение ПИД Метакона</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbExit" >
     <property name="text" >
      <string>Выход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>starterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel" >
     <x>150</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
